### DISCUSSION TITLE
Payload plugin - Cloud Storage - v1.0.0 released!

### DISCUSSION THREAD

**Discussion by richardvanbergen** [payloadcms/payload#275] [Show and tell]
Hello all,

I'm very excited to release my first ever real attempt at a community plugin or open source project. I'm happy enough with the API and the quality of the documentation and testing that I've decided to tag this as version 1.0.0 😁 👍 

I have also done some live integration testing in my own project and all the features seem to be working correctly so it's not just tested in theory according to my unit tests.

Now, as I'm sure you might have noticed I've named this plugin "cloud storage" and not "S3 storage" or something like that and I did one to release with at least one more provider but I don't think it'll take me much longer to add the code. It's just a question of the time it takes to test it.

Please let me know what you think (and be honest I can fix broken things). Thank you very much to the payload devs for the great plugin system and getting the `req.payloadUploadSizes` issue cleared up so quickly, it was so much cleaner and faster than what I was planning to do.

https://www.npmjs.com/package/payload-plugin-cloud-storage

🍾 🎉 🥳

**Comment by jmikrut** [comment]
Amazing!!!!!!

The team and I are pumped about this. We are going to use it in a project of our own coming up shortly and I could not be more excited.

@richardvanbergen — we have a ..... big email being sent out to our audience later this week. In that email, and on our social channels, we'd like to feature your work as our first ever community plugin. We'd also like to highlight it within our Plugins documentation as well.

Are you up for that? What do you think?

Reach out to us via email with your thoughts if you'd rather chat there. But overall, THANK YOU! This is awesome!

**Reply by richardvanbergen** [reply]
I am 100% up for it and I would love that, it's honestly my goal.

Though, if you could do me a personal favour before and give it a quick test? I've tested it myself but I've had no outside validation and I'm paranoid that something has been done embarrassingly wrong between then and now. I know it works I just need a little validation before it gets advertised.

**Reply by DanRibbens** [reply]
That is fair! I'll kick the tires today and reply after I've tried it.

**Reply by DanRibbens** [reply]
@richardvanbergen Great work on the plugin!
I was able to create an image and delete it using a Digital Ocean space.
The only thing I found was this minor readme change: https://github.com/richardvanbergen/payload-plugin-cloud-storage/issues/13

I didn't test out the optional getEndpoint method.

**Reply by richardvanbergen** [reply]
Thanks @DanRibbens. I'll fix up that README today.

I'm glad my testing worked out though, appreciate the feedback, makes me a lot more comfortable. Feel free to send it out!

**Reply by richardvanbergen** [reply]
> big email being sent out to our audience later this week. In that email

@jmikrut @DanRibbens FYI, your email signup at the bottom of the homepage doesn't seem to be working. Nothing get's posted in the network inspector for me.

Thought you should know.

**Reply by DanRibbens** [reply]
Thanks @richardvanbergen! It is fixed now. 👍

**Comment by Duckinm** [comment]
Really love this! Btw, I got an issue with the built-in media URL
How can I change the URL in the following images 

As I already changed the adminThumbnail to the CloudStorageUrl.
Thumbnail work fine, but copy clipboard Url still the same


![Screen Shot 2564-08-13 at 5 31 04 AM](https://user-images.githubusercontent.com/29366271/129278809-a35fd875-f188-4579-935e-343cde8ffad9.png)

**Reply by richardvanbergen** [reply]
Hey @Duckinm I think that might be one for the payload devs, as far as I'm aware there's no setting I can used to change that.

I've opened up an issue #297.

**Comment by pixexid** [comment]
its possible to add backblaze B2?
